server/server.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express');
+const session = require('express-session');
+const bodyParser = require('body-parser');
+const passport = require('passport');
+const redis = require('connect-redis')(session);
+
+const routes = {
+  board : require('./routes/board')
+}
+
+const PORT = process.env.PORT || 8080;
+const ENV = process.env.development || 'development';
+const SESSION_SECRET = process.env.SESSION_SECRET || 'keyboard cat';
+
+const app = express();
+
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
We won't need this one anymore. Everything will be through json from here on out.